### DISCUSSION TITLE
Image Thumbnails in Card View?

### DISCUSSION THREAD

**Discussion by RedMageKnight** [levito/tt-rss-feedly-theme#120] [Ideas]
Hello!

I'm not sure if this is still being worked on feature-wise, but it was this theme that actually got me to come back to using TT-RSS after having tried other solutions like NewsBlur and FreshRSS - I just really like this design and layout that was hard to emulate in the other solutions.

I had wondered if it was possible to add image thumbnails to the un-opened articles in the card view? Perhaps by taking the first image in the article and using a truncated form of that image as the thumbnail or some other transformation method to give the card view a little more pop? If not possible, no worries, was just curious of the capability. If it's something I can add as modified CSS to the existing theme or any other settings I can change or supported plugins that could do this, I'd be open to the idea, just not sure where to start.

Thank you!

**Comment by kikotomaz** [comment]
I was hoping for something similar. I know plenty of RSS feeds have a thumbnail as property of the article already, so I'm surprised it's not used.

I subscribe to lot's of photography and filmmakers so seeing the preview of the photo(s) or video thumbnail would be really nice.
